src/components/molecules/PageHeader.jsx

```diff
@@ -54,7 +54,7 @@ function PageHeader () {
           >
             <Grid item>
               <Typography variant="h5" className={styles.subTitle}>
-                Unlocking every student’s potential in math!
+                Unlocking every student’s potential in STEM
```

Comment:
Was dropping `!` intentional?

Reply:
Yes, according to the google doc, our current line summary is "Unlocking every student's potential in STEM"